workflow/controller/scope.go

```diff
@@ -14,8 +15,8 @@ type wfScope struct {
 }
 
 // replaceMap returns a replacement map of strings intended to be used simple string substitution
-func (s *wfScope) replaceMap() map[string]string {
-	replaceMap := make(map[string]string)
+func (s *wfScope) replaceMap() common.Parameters {
```

Comment:
Please rename this function `parameters()` or `getParameters()` or something similar